src/utils/index.js

```diff
@@ -109,3 +109,14 @@ export const renderExplainNode = (node) => {
     const parts = node.name.split('|');
     return parts.length > 1 ? parts[1] : node.name;
 };
+
+export const normalizeProgress = (progress) => {
```

Comment:
No need to move this function to common utils, because it is used only in one file.